### PR TITLE
Upgrading protoc-gen-star to v2

### PR DESCRIPTION
**Pull request by oxisto**
A recent upgrade to but seems to make this "options" warning an error, so its time to finally resolve it.

### CODE REVIEW

File: example/example.pb.go
Comment (ucpr) [inline example/example.pb.go:1]: Is the diff of this file the same output when generated with the latest commit?
Can you try generate again?

Comment (oxisto) [reply example/example.pb.go:1]: I am currently traveling but I can try this weekend

Comment (oxisto) [reply example/example.pb.go:1]: I re-run make gen-tag again and now two files have been changed... Should I just exclude these changes?

**Review by ucpr — APPROVED** [review]
thx!

**Review by ldepaulatrac — APPROVED** [review]

**Review by mesmerx — APPROVED** [review]

**Pull request merged**